ext/phar/tests/tar/gh16695_1.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+GH-16695 (phar:// tar parser and zero-length file header blocks)
+--CREDITS--
+hakre
```

Comment:
Have read this in CONTRIBUTING.md:

> For newly created tests, a `--CREDITS--` section should no longer be included,
as test authorship is already accurately tracked by Git. If multiple authors
should be credited, the `Co-authored-by` tag in the commit message may be used.

ref: 7514b5d91c ("Mention that zend_parse_parameters should not be tested", 2019-07-07)

at least fyi @nielsdos  , if you consider this and you need a user/email for the Co-authored-by trailer: Hans Krentel (hakre) <hanskrentel@yahoo.de> @nielsdos

Reply:
Yeah I know the CONTRIBUTING doc says that, but I find it a stupid rule because the Co-authored-by tag is not granular. (it's barely a rule imo due to the weak wording). However, a big chunk of the patch is tests of yours so I'll add a Co-authored-by tag upon merging.